### PR TITLE
Reworking the API Description format

### PR DESCRIPTION
**Pull request by automatthew**
Note that this PR compares against the `PSAD` branch, rather than `master`.  Until all the products are ready to switch over to using the new Panda Sky API Description format, we won't be merging to master.

The "root" PR is here: https://github.com/pandastrike/panda-sky/pull/93

This PR features the bare minimum required to get the Blurb9 PSAD to produce exactly the same CloudFormation Template as before.

### CODE REVIEW

File: test/data/blurb9/_api_config.yaml [hunk 1]
```diff
@@ -2,11 +2,9 @@ resources:
   discovery:
     path: /
     description: Provides a directory of available resources
-    actions:
-      get:
+    methods:
+      GET:
         method: GET
```
Comment (maximbert) [inline test/data/blurb9/_api_config.yaml]: Any reason to leave the method here? It seems from all other examples that the top key is sufficient.

File: test/data/blurb9/_api_config.yaml [hunk 2]
```diff
@@ -46,14 +44,9 @@ resources:
   blurbs:
     path: blurbs
     description: Virtual collection of blurbs
-    actions:
-      create:
+    methods:
+      POST:
         method: POST
```
Comment (maximbert) [inline test/data/blurb9/_api_config.yaml]: ditto

File: test/data/blurb9/_api_config.yaml [hunk 3]
```diff
@@ -107,14 +100,9 @@ resources:
   blurb:
     path: 'blurbs/{key}'
     description: Individual blurbs
-    actions:
-      get:
+    methods:
+      GET:
         method: GET
```
Comment (maximbert) [inline test/data/blurb9/_api_config.yaml]: ditto

File: test/data/blurb9/_api_config.yaml [hunk 4]
```diff
@@ -158,12 +146,8 @@ resources:
               method.response.header.Access-Control-Allow-Headers: true
               method.response.header.Access-Control-Allow-Methods: true
               method.response.header.Access-Control-Allow-Origin: true
-      update:
+      PUT:
         method: PUT
```
Comment (maximbert) [inline test/data/blurb9/_api_config.yaml]: ditto

File: test/data/blurb9/_api_config.yaml [hunk 5]
```diff
@@ -205,12 +189,8 @@ resources:
               method.response.header.Access-Control-Allow-Headers: true
               method.response.header.Access-Control-Allow-Methods: true
               method.response.header.Access-Control-Allow-Origin: true
-      delete:
+      DELETE:
         method: DELETE
```
Comment (maximbert) [inline test/data/blurb9/_api_config.yaml]: ditto

**Comment by maximbert** [comment]
LGTM. I've only pointed to minor changes on examples, PR can be merged.

File: test/data/blurb9/_api_config.yaml (hunk 1 above)
Comment (automatthew) [inline test/data/blurb9/_api_config.yaml]: good catch.  I'll have to check.

Comment (automatthew) [inline test/data/blurb9/_api_config.yaml]: This is the config object used to populate the api.yaml template, so its contents don't matter now unless they affect the outcome (and it's a pain to change them right now)

**Pull request merged**